### PR TITLE
docs: update changelog

### PR DESCRIPTION
**Pull request by lumianph**

### CODE REVIEW

**Comment by dl239** [comment]
the following files also need to update
- docs/en/about/release_notes.md
- docs/zh/about/release_notes.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,31 +2,35 @@
...
-- Update catalog if leade role changed (#1655 @dl239)
-- Fix unsaferow relavent problems(#1298, #1312, #1326, #1362, #1637, #1381, #1731 @tobegit3hub)
-- Query result is incorrect after adding a new index in standalone mode (#1721 @keyu813)
-- Fail to get correct result for `SHOW JOBS` command(#1453 @tobegit3hub)
-- Result is incorrect for date columns with `UnsafeRowOpt`(#1469 @tobegit3hub)
-- Other minor bug fix (#1698 @kfiring; #1651 @kutlayacar; #1621 @KaidoWang; #1150, #1243 @tobegit3hub; )
+- Incorrect results when the order of conditions specified in `where` is different from that of the index (#1709 @aceforeverd)
+- Incorrect results of `lag`/`at`/`lead` under certain circumstances (#1605 #1739 @aceforeverd)
+- Memory leakage in `zk_client` (#1660 @wuxiaobai24)
+- Catalog update failed if the role of a tablet is changed (#1655 @dl239)
```
Comment (dl239) [inline CHANGELOG.md]: Catalog have not updated

**Comment by dl239** [comment]
I have modified the [milestones](https://github.com/4paradigm/OpenMLDB/pull/1755/files#diff-9b525f33653069287b53558b59c002eeb96fbde4c04f26b8790568a60e9e1f7f) in #1755 . it also need to check

**Comment by lumianph** [comment]
> I have modified the [milestones](https://github.com/4paradigm/OpenMLDB/pull/1755/files#diff-9b525f33653069287b53558b59c002eeb96fbde4c04f26b8790568a60e9e1f7f) in #1755 . it also need to check

this change is conflict with my another change (not pushed yet), will handle this later

**Review by dl239 — APPROVED** [review]

**Pull request merged**